### PR TITLE
here u go :troll:

### PR DESCRIPTION
**Pull request by 40j**

### CODE REVIEW

**Comment by 40j** [comment]
added cock.li emails and protonmail
:trollface:

File: src/Utility/Constants.ts [hunk 1]
```diff
@@ -196,6 +196,41 @@ export const allowedEmails = [
   'graduate.org',
   'musician.org',
   'teachers.org',
+  'shiba.bar',
+  'protonmail.com',
+  'aol.com',
+  'mailcow.email',
+  'cock.li',
+  'discord.com',
```
Comment (Kurpp) [inline src/Utility/Constants.ts:204]:
```suggestion

```
unrealisitic

File: src/Utility/Constants.ts [hunk 2]
```diff
@@ -196,6 +196,41 @@ export const allowedEmails = [
...
+  'googlemail.com',
+  'yahoo.com',
```
Comment (Kurpp) [inline src/Utility/Constants.ts:206]:
```suggestion

```
duplicate

File: src/Utility/Constants.ts [hunk 3]
```diff
@@ -196,6 +196,41 @@ export const allowedEmails = [
...
+  'cock.email',
+  'firemail.cc',
+  'memeware.net',
+  'cocaine.ninja',
+  'waifu.club',
+  'dicksinhisan.us',
+  'loves.dicksinhisan.us',
+  'wants.dicksinhisan.us',
+  'dicksinmyan.us',
+  'loves.dicksinmyan.us',
```
Comment (Kurpp) [inline src/Utility/Constants.ts:232]:
```suggestion
  'goat.si',
  'national.shitposting.agency',
  'tfwno.gf',
  'firemail.cc',
  'memeware.net',
  'waifu.club',
```

drop the inappropriate emails

File: src/Utility/Constants.ts [hunk 4]
```diff
@@ -196,6 +196,41 @@ export const allowedEmails = [
   'graduate.org',
   'musician.org',
   'teachers.org',
+  'shiba.bar',
+  'protonmail.com',
+  'aol.com',
+  'mailcow.email',
+  'cock.li',
```
Comment (Kurpp) [inline src/Utility/Constants.ts:203]:
```suggestion

```

innap

**Review by Kurpp — CHANGES_REQUESTED** [review]
few email drops and we should be good

File: src/Utility/Constants.ts [hunk 5]
```diff
@@ -196,6 +196,41 @@ export const allowedEmails = [
...
+  'discord.com',
+  'googlemail.com',
+  'yahoo.com',
+  'zoho.com',
+  'zohomail.com',
+  'privatemail.com',
+  'ionos.com',
```
Comment (Kurpp) [inline src/Utility/Constants.ts:207]:
```suggestion

```

unused/paid mail serivce

**Review by Kurpp — CHANGES_REQUESTED** [review]

**Review by Kurpp — APPROVED** [review]

**Pull request merged**